### PR TITLE
Task/category landing govau 354

### PR DESCRIPTION
**Pull request by simonschwartz**
The PR updates the layout of the category listing page. This closes [GOVAU-354]

It also adds the expand all and collapse all functionality to accordion groups. This is achieved using Javascript. Consideration has been given to users that may have Javascript disabled or are using an older version of IE. If JS is disabled the accordion group toggle functionality is not added to the DOM but users can continue to operate the accordions individually.

The Javascript is currently using a shim to ensure compatability with IE9 as `.classList` is not supported. This may be able to be refactored out in the future.

accordions

### CODE REVIEW

**Comment by simonschwartz** [comment]
Update Changelog

File: app/assets/images/chevron-black.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (maxious) [inline app/assets/images/chevron-black.svg]: Would this work just as well with https://github.com/AusDTO/gov-au-beta/blob/develop/vendor/assets/images/icons/chevron-down--navy.svg ?

File: app/assets/stylesheets/_category-accordions.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  summary {
+    background-color: $background-secondary-colour;
+    border-bottom: 1px solid $border-colour;
+    border-top: 1px solid $border-colour;
+    color: $body-text-colour;
+    font-size: rem(20);
+
+    &::before {
+      background-image: asset-url('chevron-black.svg');
```
Comment (maxious) [inline app/assets/stylesheets/_category-accordions.scss]: https://github.com/AusDTO/gov-au-beta/blob/develop/vendor/assets/stylesheets/_ui-kit.scss#L5130 includes PNG fall back too.

File: app/assets/images/chevron-black.svg (hunk 1 above)
Comment (simonschwartz) [inline app/assets/images/chevron-black.svg]: Unfortunately not, I checked and the designers need the icon to be black for these types of accordions.

File: app/assets/stylesheets/_category-accordions.scss (hunk 2 above)
Comment (simonschwartz) [inline app/assets/stylesheets/_category-accordions.scss]: Good point I'll add a png fallback for the black icon

**Comment by maxious** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

**Pull request merged**